policyengine_us/parameters/gov/states/nm/tax/income/exemptions/blind_and_aged/age_threshold.yaml

```diff
@@ -0,0 +1,13 @@
+description: New Mexico minimum age to qualify for aged or blind exemption.
+values:
+  1987-01-01: 65
+metadata:
+  label: New Mexico aged and blind exemption age threshold
+  period: year
+  unit: year
+  reference:
+    - title: 7-2-5.2. EXEMPTION--INCOME OF PERSONS SIXTY-FIVE AND OLDER OR BLIND
+      href: https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/856ebf4b-3814-49dd-8631-ebe579d6a42b/Personal%20Income%20Tax.pdf
```

Comment:
could you link the pages to the pdfs

Reply:
all yaml files and references

Reply:
also please add `[this](https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!fragment/zoupio-_Toc140503666/BQCwhgziBcwMYgK4DsDWszIQewE4BUBTADwBdoAvbRABwEtsBaAfX2zgEYAWABgFYeAZgBsogJQAaZNlKEIARUSFcAT2gBydRIiEwuBIuVrN23fpABlPKQBCagEoBRADKOAagEEAcgGFHE0jAAI2hSdjExIA)` source